content/docs/render-props.md

```diff
@@ -42,22 +42,22 @@ class MouseTracker extends React.Component {
   render() {
     return (
       <div style={{ height: '100%' }} onMouseMove={this.handleMouseMove}>
-        <h1>Move the mouse around!</h1>
-        <p>The current mouse position is ({this.state.x}, {this.state.y})</p>
+        <h1>Porusz myszką!</h1>
+        <p>Aktualna pozycja myszki to ({this.state.x}, {this.state.y})</p>
```

Comment:
```suggestion
        <p>Aktualna pozycja kursora to ({this.state.x}, {this.state.y})</p>
```
Trzeba rozróżniać. Myszka to urządzenie i jego pozycji raczej nie określamy. Te współrzędne dotyczą kursora na ekranie.

Reply:
Tłumaczyłem jako "Myszka" ponieważ komponent używany w przykładzie jest tak nazywany. :P Ale nie ma problemu może być i kursor. :)